elements/pfe-sass/mixins/_mixins.scss

```diff
@@ -192,6 +193,156 @@
   }
 }
 
+/// ===========================================================================
+/// Tab-specific SASS mixins
+/// ===========================================================================
+@mixin pfe-tabs-vertical {
+  padding-top: #{pfe-var(container-padding)};
+}
+
+@mixin pfe-tabs-tabset {
+  --pfe-tabs--BorderColor: #{pfe-color(surface--border)};
```

Comment:
Need to use the same approach here that we did for the accordion - passing in the component name as a variable so that the custom property is component-scoped.

Reply:
@castastrophe I moved all tab mixins (except for one) back into the tab scss files so I did not do this.